persistence/src/main/java/edu/unc/lib/dl/persist/services/importxml/ImportXMLRequest.java

```diff
@@ -30,6 +33,7 @@
  */
 public class ImportXMLRequest {
     private String userEmail;
+    @JsonDeserialize(as = AccessGroupSetImpl.class)
```

Comment:
This tells jackson to instantiate an AccessGroupSetImpl object for this instance variable, since the variable itself is declared as an interface, so it needs to know what implementation to use